code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -287,130 +287,6 @@
 	H.put_in_hands(new /obj/item/claymore(H), TRUE)
 	H.equip_to_slot_or_del(new /obj/item/spear(H), ITEM_SLOT_BACK)
 
-
-/obj/item/voodoo
-	name = "wicker doll"
-	desc = "Something creepy about it."
-	icon = 'icons/obj/wizard.dmi'
-	icon_state = "voodoo"
```

Comment:
![image](https://user-images.githubusercontent.com/24975989/111424532-d2f97d80-86e9-11eb-90da-67e34c5d68b7.png)

Doesn't look like anything else uses this icon_state. Worth doing a sweep over the icon files to clear it out as well? Any removed icons can always be pushed to the other icon repo or retrieved thanks to Github in general.

Reply:
What you're referring to as github is actually git +gibhub, or as I've taken to calling it, gitgub

Reply:
Gitgub also gets aliased to Gitgud too.